atlas/map.go

```diff
@@ -187,21 +191,40 @@ func (m Map) Encode(ctx context.Context, tile *slippy.Tile) ([]byte, error) {
 					}
 				}
 
+				defer func() {
```

Comment:
Can you add some code comments around this block? We migth also want to put move this to a separate function since this is debugging code and the Encode() func is already quite long.